apps/site/src/app/(home)/sections/MentorVolunteer/Stand.tsx

```diff
@@ -0,0 +1,50 @@
+import type { ComponentProps } from "react";
+import Image from "next/image";
+import Link from "next/link";
+import Button from "@/lib/components/Button/Button";
+import standBottom from "./stand-bottom.png";
+
+interface StandProps {
+	imageSrc: ComponentProps<typeof Image>["src"];
+	header: string;
+	description: string;
+	buttonText: string;
+	buttonHref: ComponentProps<typeof Link>["href"];
+}
+
+const Stand: React.FC<StandProps> = ({
+	imageSrc,
+	header,
+	description,
+	buttonText,
+	buttonHref,
+}) => {
+	return (
+		<div className="max-w-[450px] text-center flex flex-col items-stretch">
+			<div className="relative aspect-[1/0.7] z-10">
+				<h2 className="absolute mt-[4.5%] text-2xl z-20 w-full text-[#FFDA7B]">
```

Comment:
Suggestion: Might be worth adding the colors we use in Tailwind classes as color variables in our Tailwind config.

Reply:
I was actually working on designing a set of color scales that match the colors in the Figma, however that's probably outside the scope of this issue.